.github/workflows/agw-workflow.yml

```diff
@@ -474,7 +474,13 @@ jobs:
       - uses: actions/checkout@v2
       - uses: DoozyX/clang-format-lint-action@v0.12
```

Comment:
I'm not sure about the status collection / posting to Slack for this workflow given the two clang-format phases. Would need DevOps advice on how this will behave / how to amend to allow collection of aggregate failures of two clang format linters.

Reply:
Since this check is required, I'm not sure if we need a slack notification for

Reply:
Agree